component-catalog/src/Examples/Table.elm

```diff
@@ -125,16 +125,16 @@ example =
                 ]
             , case ( showHeader, isLoading ) of
                 ( True, False ) ->
-                    Table.view { additionalStyles = [], alternatingRowColors = alternatingRowColors } columns data
+                    Table.view [] columns data
 
                 ( False, False ) ->
-                    Table.viewWithoutHeader { additionalStyles = [], alternatingRowColors = alternatingRowColors } columns data
+                    Table.viewWithoutHeader [] columns data
 
                 ( True, True ) ->
-                    Table.viewLoading { additionalStyles = [], alternatingRowColors = alternatingRowColors } columns
+                    Table.viewLoading [] columns
 
                 ( False, True ) ->
-                    Table.viewLoadingWithoutHeader { additionalStyles = [], alternatingRowColors = alternatingRowColors } columns
+                    Table.viewLoadingWithoutHeader [] columns
```

Comment:
I think we should conditionally add the `disableAlternatingRowColors` attribute here! The toggle isn't doing anything in the demo anymore 😅.

https://github.com/user-attachments/assets/e380244e-0c5f-48ad-883c-4f8a5df30f26

Reply:
lol, thank you for noticing